mivs/model_checks.py

```diff
@@ -18,7 +18,7 @@ def _is_invalid_url(url):
 
 @validation.IndieStudio
 def new_studio_deadline(studio):
-    if studio.is_new and c.AFTER_ROUND_ONE_DEADLINE:
+    if studio.is_new and c.AFTER_ROUND_ONE_DEADLINE and c.HAS_INDIE_ADMIN_ACCESS:
```

Comment:
I think this should be ``not c.HAS_INDIE_ADMIN_ACCESS``, right?  As it is now, it will ONLY return this error for logged in admins instead of returning it for everyone else.